docs/spark.rst

```diff
@@ -0,0 +1,74 @@
+Running Spark on Toil
+=====================
+
```

Comment:
Notes from my experience:
- Remember to mention docker-machine is on os-x
- Remember to mention that they will need to spin up a docker daemon if on a single machine:
 I'd link to this tutorial: https://docs.docker.com/machine/get-started/
- Point out that they will need spark installed and need to put pySpark on their python path
- We need to incorporate py4j into the virtualenv, probably installed as an optional extra with a "spark" flag.